src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -115,6 +115,19 @@ def make_parameter_file(out_path: Path, reference_scene: str, secondary_scene: s
         json.dump(output, f)
 
 
+def make_browse_image(input_tif: str, output_png: str) -> None:
```

Comment:
Since this is a function other HyP3-ISCE2 workflows will use, I'd recommend moving it to `src/hyp3_isce2/utils.py`.